docs/monitoring-usage.rst

```diff
@@ -278,3 +278,86 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation for the `Creating Custom Metrics`_ for more information.
+
+To create a custom :class:`~gcloud.monitoring.timeseries.TimeSeries` value, you must first create a
+custom :class:`~gcloud.monitoring.metric.MetricDescriptor` object, and then use its
+type to create a fully parameterized :class:`~gcloud.monitoring.metric.Metric` by providing
+specific values for the available labels. For more information on creating metric descriptors,
+see the `Metric Descriptors`_ section.
+
+You must also create a fully parameterized :class:`~gcloud.monitoring.resource.Resource` from one
+of the available ``ResourceDescriptors`` which can be listed using the
+:class:`~gcloud.monitoring.resource.ResourceDescriptor` class, which similarly requires that its
+available labels have values specified. The ``project-id`` label is a special case that should
+not be specified.
+
+>>> from gcloud import monitoring
+>>> # Create a Resource by its resource name, assign labels as dict
+>>> resource = client.resource('gce_instance', labels={
+...     'instance_id': '1234567890123456789',
+...     'zone': 'us-central1-f'
+... })
+>>> # Create a Metric using its metric descriptor. assigning label as dict
+>>> metric = client.metric(type='custom.googleapis.com/my_metric', labels={
+...      'status': 'successful'
+... })
+
+Please refer to the `Metrics`_ documentation for more information.
+
+With a ``Metric`` and ``Resource`` specified, the :class:`~gcloud.monitoring.client.Client`
+can be used to write :class:`~gcloud.monitoring.timeseries.Point` values.
+
+When writing points, the Python type of the value must match the *value_type* specified
+in the associated :class:`~gcloud.monitoring.metric.MetricDescriptor`. For example, a Python
```

Comment:
Since code for writing custom metrics might not need to deal with `MetricDescriptor` objects, I think it would be better to link to the API documentation:

...must match the value type specified in the appropriate [metric descriptor](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors). For example, ...